### ISSUE TITLE
Add more details about the screenshot in readme

### ISSUE THREAD

**Issue by JingBh** [imbushuo/EnergyStar#18]
![the image in readme](https://camo.githubusercontent.com/be819089d5b958d18930eb238a29563235c3aaba9c7794a38e9ffaac31eaca3c/68747470733a2f2f7062732e7477696d672e636f6d2f6d656469612f4659356f416946565141416c5235643f666f726d61743d706e67266e616d653d6d656469756d)

Is it possible to add more information about the device on which this battery life data was measured, such as CPU/GPU specs.

I think it is not possible to get 10+ hours of battery life on high-end mobile platforms, such as an i9-12900H, which has a base power of 45W.

**Comment by hez2010** [comment]
As per https://github.com/imbushuo/EnergyStar/blob/18872289ac4de7d22208f1fc5b156abaa904dff8/EnergyStar/Program.cs#L36, I think it's ThinkPad Z13.

**Comment by JasonWei512** [comment]
... maybe someone can open an issue to let users share their battery life before and after using this utility for comparison?

**Comment by JingBh** [comment]
> ... maybe someone can open an issue to let users share their battery life before and after using this utility for comparison?

Yeah, they should at least make a before-after comparison in the readme.
